peakElem.py

```diff
@@ -0,0 +1,21 @@
+# Couldn't find the solution for when there is no peak Eg : [8,7,5,2,5,7,8]
+# Didn't understand meaning of nums[-1] = nums[n] = -∞
```

Comment:
basically we are saying is that for [8,7,5,2,5,7,8] we are assuming that the array is [0,8,7,5,2,5,7,8,0] that for the egde values the left and right value is lower